pyspectre/core.py

```diff
@@ -2,12 +2,14 @@
 
 import os
 import re
+import subprocess
```

Comment:
What is the reason for this import? the line below should get everything we need from `subprocess`.